### PR TITLE
cam_cesm2_1_rel_52: Fix COSP snow optics bug, and nhtfrq namelist definition.

### LINKED ISSUE #262: bug in the lidar simulator (COSP1.4 and COSP2)
Greg Cesana from GISS/Columbia Univ. recently reported a bug in the lidar simulator in COSP1.4 and COSP2. For COSP1.4, “snow” has been mistakenly added to liquid in COSP1.4. For COPS2, “snow” is not accounted for at all.  This will have potentially large impact on cloud phase diagnostics from the lidar simulator. So we need to fix it.

For COSP2, to fix the bug, we only need to change the line 463-468 in cosp_optics.F90 

from 
       betatot_ice(1:npoints,icol,1:nlev) = betatot_ice(1:npoints,icol,1:nlev)+ &
            kp_part(1:npoints,1:nlev,INDX_LSICE)*alpha_part(1:npoints,1:nlev,INDX_LSICE)+ &
            kp_part(1:npoints,1:nlev,INDX_CVICE)*alpha_part(1:npoints,1:nlev,INDX_CVICE)
       tautot_ice(1:npoints,icol,1:nlev) = tautot_ice(1:npoints,icol,1:nlev)  + &
            tau_part(1:npoints,1:nlev,INDX_LSICE) + &
            tau_part(1:npoints,1:nlev,INDX_CVICE)
 
 to 
       betatot_ice(1:npoints,icol,1:nlev) = betatot_ice(1:npoints,icol,1:nlev)+ &
            kp_part(1:npoints,1:nlev,INDX_LSICE)*alpha_part(1:npoints,1:nlev,INDX_LSICE)+ &
            kp_part(1:npoints,1:nlev,INDX_CVICE)*alpha_part(1:npoints,1:nlev,INDX_CVICE)+ &
            kp_part(1:npoints,1:nlev,INDX_LSSNOW)*alpha_part(1:npoints,1:nlev,INDX_LSSNOW)
       tautot_ice(1:npoints,icol,1:nlev) = tautot_ice(1:npoints,icol,1:nlev)  + &
            tau_part(1:npoints,1:nlev,INDX_LSICE) + &
            tau_part(1:npoints,1:nlev,INDX_CVICE) + &
            tau_part(1:npoints,1:nlev,INDX_LSSNOW)
 
For COSP1.4, we need to change the line 484-491 in lidar_simulator.F90
 
from 
          DO i = 2, npart,2
               betatot_ice(:,:) = betatot_ice(:,:)+ kp_part(:,:,i)*alpha_part(:,:,i)
               tautot_ice(:,:) = tautot_ice(:,:)  + tau_part(:,:,i)
          ENDDO ! i
          DO i = 1, npart,2
               betatot_liq(:,:) = betatot_liq(:,:)+ kp_part(:,:,i)*alpha_part(:,:,i)
               tautot_liq(:,:) = tautot_liq(:,:)  + tau_part(:,:,i)
          ENDDO ! i
 
to
          DO i = 2, npart,2
               betatot_ice(:,:) = betatot_ice(:,:)+ kp_part(:,:,i)*alpha_part(:,:,i)
               tautot_ice(:,:) = tautot_ice(:,:)  + tau_part(:,:,i)
          ENDDO ! i
               betatot_ice(:,:) = betatot_ice(:,:)+ kp_part(:,:,5)*alpha_part(:,:,5)
               tautot_ice(:,:) = tautot_ice(:,:)  + tau_part(:,:,5)
          DO i = 1, npart-1,2
               betatot_liq(:,:) = betatot_liq(:,:)+ kp_part(:,:,i)*alpha_part(:,:,i)
               tautot_liq(:,:) = tautot_liq(:,:)  + tau_part(:,:,i)
          ENDDO ! i

Yuying Zhang (LLNL)

### LINKED ISSUE #265: Need to update "nhtfrq" namelist definition
Currently the definition for the namelist variable `nhtfrq` states that only the first file can contain monthly averages.  However, this is not actually the case, and appears to be causing some confusion among certain users.  Thus the namelist variable definition should be updated to remove this "first file" statement.

### PR DESCRIPTION
**Pull request by nusbaume**
Fixes #262 
Fixes #265 

This PR also changes answers for COSP, so I am having two scientists double-check the code change.

### CODE REVIEW

**Comment by nusbaume** [comment]
@zyuying and @brianpm this pull request changes answers for the COSP outputs.  Can you please verify that this is expected, and also make sure that my code changes to `cosp_optics.F90` are correct?  Thank you!

**Comment by brianpm** [comment]
For sure the COSP output should change. It should only impact the CALIPSO / lidar simulator output, I think.

**Comment by zyuying** [comment]
@nusbaume , Your code changes are correct. As @brianpm mentioned, it will only change the lidar simulator output.

**Comment by nusbaume** [comment]
Thanks!  I verified that only the CALIPSO outputs change, so I believe this PR is ready for review from the SEs.

File: doc/ChangeLog [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+
+izumi/nag:
+
+024 bl317 TBL.sh f10c5aqcdm outfrq3s_cosp 9s -> COSP outputs differ
+027 bl318 TBL.sh f10c6aqcdm sat_hist 9s      -> COSP outputs differ
+
+izumi/pgi: ALL PASS
+
+CAM tag used for the baseline comparison tests if different than previous
+tag: cam_cesm2_1_rel_48
```
Comment (gold2718) [inline doc/ChangeLog:59]: Is there a reason you are not using the cam_cesm2_1_rel_51 baselines?

Comment (nusbaume) [reply doc/ChangeLog:59]: Sorry, I did compare to to the `cam_cesm2_1_rel_51` baselines.  I simply forgot to update the ChangeLog (which I have done now).

**Review by cacraigucar — APPROVED** [review]

**Review by gold2718 — APPROVED** [review]

**Review by brian-eaton — APPROVED** [review]

**Pull request merged**